node_modules/oae-core/about/js/about.js

```diff
@@ -0,0 +1,94 @@
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
+
+    // When this widget is loaded, the profile representing the context for which the about widget
+    // needs to be rendered should be passed in as `widgetData.context`. Further updates to the
+    // profile in context should be made by issuing the document event `oae.about.update-profile`
+    // with the updated profile as the first parameter
+    return function(uid, showSettings, widgetData) {
+
+        // The widget container
+        var $rootel = $('#' + uid);
+
+        // The context profile (e.g., group) to use for rendering
+        var contextProfile = widgetData.context;
+
+        /**
+         * Render the header of the about widget
+         */
+        var setUpAboutHeader = function() {
+            var action = null;
+
+            if (contextProfile.isManager) {
+                action = {
+                    'icon': 'fa-pencil',
+                    'trigger': 'group-trigger-editgroup',
+                    'data': {
+                        'editgroup-focus': '#editgroup-description'
+                    }
+                };
+            }
+
+            oae.api.util.template().render($('#about-header-template', $rootel), {'action': action}, $('#about-header', $rootel));
+        };
+
+        /**
+         * Render the profile description of the about widget
+         */
+        var setUpAboutDescription = function() {
+            oae.api.util.template().render($('#about-description-template', $rootel), {'contextProfile': contextProfile}, $('#about-description', $rootel));
+        };
+
+        /**
+         * Render the members of the about widget
+         */
+        var setUpAboutMembers = function() {
+            var data = {
+                'contextProfile': contextProfile,
+                'thumbnails': _.chain(contextProfile.members)
+                    .pluck('profile')
+                    .map(function(member) {
+                        return {
+                            'entity': member,
+                            'displayOptions': {
+                                'addVisibilityIcon': false,
+                                'size': 'medium'
+                            }
+                        };
+                    })
+                    .value()
+            };
+
+            oae.api.util.template().render($('#about-members-template', $rootel), data, $('#about-members', $rootel));
+        };
+
+        /**
+         * Re-render the about description content when the group profile updates
+         */
+        $(document).on('oae.about.profile-update', function(ev, updatedContextProfile) {
```

Comment:
It would be good to not make this an event specific to this widget that then needs to be added in other widgets. A more general event like `oae.context.update` (cfr `'oae.context.get'`) would be more appropriate